app/controllers/posts_controller.rb

```diff
@@ -65,6 +65,6 @@ def load_post
 
     flash[:alert] = 'Post Not found!' if @post.nil?
 
-    redirect_to posts_path
+    redirect_to posts_path if @post.nil?
```

Comment:
May as well collapse this into a single if block

Reply:
I had it like that in the past and got complaints from rubocop and RubyMine